tests/Maker/MakeUserTest.php

```diff
@@ -37,6 +37,21 @@ public function getTestDetails()
             ->updateSchemaAfterCommand(),
         ];
 
+        yield 'user_security_entity_with_password_authenticated_user_interface' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeUser::class),
+            [
+                // user class name
+                'User',
+                'y', // entity
+                'email', // identity property
+                'y', // with password
+                'n', // legacy password interface in >5.3
```

Comment:
What question is the answers?

Reply:
umm good question.. think that was for an earlier WIP that i reverted.. 

removed